095-add-support-volumeattributesclassname.md

```diff
@@ -0,0 +1,102 @@
+# Add `volumeAttributesClassName` to the storage configuration
+
+Add the ability to set `volumeAttributesClassName` in the configuration for `PersistentClaimStorage`.
```

Comment:
Someone reading could not understand what `PersistentVolumeClaim` is here. Of course, it's part of the Strimzi api, it's one of the model classes but need to be clarified.